### PR TITLE
Fix width of dossier and title column in meeting tab

### PR DESCRIPTION
**Pull request by tarnap**
column_width_meeting_tab

Resolves https://github.com/4teamwork/gever/issues/159

### CODE REVIEW

**Comment by deiferni** [comment]
_(squash the commits plz 😉 )_

File: opengever/meeting/tabs/meetinglisting.py [hunk 1]
```diff
@@ -81,7 +82,7 @@ class MeetingListingTab(BaseListingTab):
         {'column': 'dossier',
          'column_title': _(u'dossier', default=u'Dossier'),
          'transform': dossier_link_or_title,
-         'sortable': False},
```
Comment (phgross) [inline opengever/meeting/tabs/meetinglisting.py]: you drop the `sortable:false` here, why`

Comment (tarnap) [reply opengever/meeting/tabs/meetinglisting.py]: @phgross: do I really drop it?

Comment (phgross) [reply opengever/meeting/tabs/meetinglisting.py]: Ah you re add it in a later PR ...

**Comment by tarnap** [comment]
@phgross: As far as I was told, the custom settings will remain. These newly added widths are default values (if no custom value was set so far)

**Comment by deiferni** [comment]
@phgross no that is the idea, actually. if a new user opens that table for the first time to column widths are almost certainly too small. we attempt to find a better setting, which may just work and which will give a better first impression 😉 .

**Pull request merged**